.github/actions/next-stats-action/src/prepare/repo-setup.js

```diff
@@ -67,69 +81,81 @@ module.exports = (actionInfo) => {
         throw err
       }
 
-      for (const pkg of pkgs) {
-        const pkgPath = path.join(repoDir, 'packages', pkg)
-        const packedPkgPath = path.join(pkgPath, `${pkg}-packed.tgz`)
-
-        const pkgDataPath = path.join(pkgPath, 'package.json')
-        if (!fs.existsSync(pkgDataPath)) {
-          require('console').log(`Skipping ${pkgDataPath}`)
-          continue
-        }
-        const pkgData = require(pkgDataPath)
-        const { name } = pkgData
-
-        pkgDatas.set(name, {
-          pkgDataPath,
-          pkg,
-          pkgPath,
-          pkgData,
-          packedPkgPath,
+      await Promise.all(
+        pkgs.map(async (pkg) => {
+          const pkgPath = path.join(repoDir, 'packages', pkg)
+          const packedPkgPath = path.join(pkgPath, `${pkg}-packed.tgz`)
+
+          const pkgDataPath = path.join(pkgPath, 'package.json')
+          if (fs.existsSync(pkgDataPath)) {
+            const pkgData = JSON.parse(await fsp.readFile(pkgDataPath))
```

Comment:
We don't have to explicitly specify `encoding: utf-8` here, as per ES spec the `JSON.parse` should convert the `input` to string before parsing, and Node.js `Buffer` also uses UTF-8 sequence internally.